### ISSUE TITLE
CrossWoC Contribution

### ISSUE THREAD

**Issue by sanidhyakr18** [ritwikranjan/College-Community#2]
Hey,
Can I contribute to this project?

**Comment by seemantaggarwal** [comment]
Yes, you can. Any issue you find, will be assigned points according to the contribution

**Comment by chiragagrawal9200** [comment]
Hey  @seemantaggarwal ....I am cross woc participant and I would like to contribute to the project
